src/ui/playblast_ui.py

```diff
@@ -14,22 +22,44 @@ def get_maya_window():
     main_window_ptr = OpenMayaUI.MQtUtil.mainWindow()
     return wrapInstance(int(main_window_ptr), QWidget)
 
+class IntegerValidator(QValidator):
+    def validate(self, input_text, pos):
+        if input_text == "":
+            return QValidator.Intermediate
+        try:
+            int(input_text)
+            return QValidator.Acceptable
+        except ValueError:
+            return QValidator.Invalid
 
 class PlayblastManagerUI(QMainWindow):
 
-    def __init__(self, parent=get_maya_window()):
+    def __init__(self, parent=get_maya_window(), debug = False):
+        """
+        This function creates the main window of the UI
+        Args:
+            parent: This function is son of a previous function that allows maya to show the window
+            debug: placed to see the temporary folder of all the assets or don't see it
+        """
```

Comment:
```suggestion
        Args:
             parent (QWidget): Parent widget for the window. Default to Maya window.
            debug (bool): Initialize in debug mode. If True, the temporary folder will not be deleted.
        """
```

No te olvides del "type" de cada argumento. Te puse un ejemplo de como describiria yo estos argumentos, no queremos que sean muy especificos ya que son relativamente genericos, por ejemplo el "parent" puede ser algo que no sea necesariamente la ventana de Maya.